Entity properties display

demshy opened the pull request:
Test with https://github.com/plezanje-net/api/pull/105

Don't forget to run codegen

Route: http://localhost:4200/plezalisce/voglberg-juzna-stena/smer/wachauer-grat
Ice fall: http://localhost:4200/alpinizem/slapovi/slap/desni-slap-pod-belo-skalo
Crag: http://localhost:4200/plezalisce/seebichlhof-kraig

salamca commented:
Difficulty and extGrade should probably have different labels. They are both 'težavnost' now.

Equipment property name has a typo 'Potrebnama'
http://localhost:4200/alpinizem/slapovi/slap/moonwalker

accessTime should probably have units displayed. Are they hours?
http://localhost:4200/alpinizem/stena/bovski-gamsovec-juzna-stena

some photo links are relative, hence broken
http://localhost:4200/alpinizem/stena/bovski-gamsovec-juzna-stena

should we also add nofollow, noreferrer, noopener to all external links?

access has already been migrated to it's own property and is now duplicated
http://localhost:4200/plezalisce/colnisce

might belong to another issue but since we are working on this content:
some routes have descriptions that are not shown anywhere...
they are present in xml and also in own field in route entity
http://localhost:4200/plezalisce/misja-pec/smer/cona-udobja

demshy commented:
> Difficulty and extGrade should probably have different labels. They are both 'težavnost' now.

Until we figure out the naming for this, I have hidden our grade in case extGrade is present

> Equipment property name has a typo 'Potrebnama' http://localhost:4200/alpinizem/slapovi/slap/moonwalker

fixed

> accessTime should probably have units displayed. Are they hours? http://localhost:4200/alpinizem/stena/bovski-gamsovec-juzna-stena

this is a problem with property type no longer being time (but a normal string)), because there were some absurd values in there. I guess we will have to rename the property name to Trajanje dostopa (h)

> some photo links are relative, hence broken http://localhost:4200/alpinizem/stena/bovski-gamsovec-juzna-stena

I have replaced them with our hardcoded image links

> should we also add nofollow, noreferrer, noopener to all external links?

done

> access has already been migrated to it's own property and is now duplicated http://localhost:4200/plezalisce/colnisce

I have excluded it from migration, let's stick to our own property on crag entity

> might belong to another issue but since we are working on this content: some routes have descriptions that are not shown anywhere... they are present in xml and also in own field in route entity http://localhost:4200/plezalisce/misja-pec/smer/cona-udobja

we can do this when we add some stuff to the route page

salamca commented:
> > accessTime should probably have units displayed. Are they hours? http://localhost:4200/alpinizem/stena/bovski-gamsovec-juzna-stena
> 
> this is a problem with property type no longer being time (but a normal string)), because there were some absurd values in there. I guess we will have to rename the property name to Trajanje dostopa (h)

Values in database seem to only be numbers. (although some have decimal dot and some have decimal comma). It seems that they could be converted to numbers.
`SELECT * FROM crag_property WHERE "propertyTypeId" = 'accessTime'`
